hpi-swt2/lecture-portal
# Feature/#360 create polls in lecture created status

# Resolves #360 

This PR allows a lecturer to create pulls even before the day of the lecture while start, stop and vote functionality is still limited to the day of the lecture.

Acceptance Criteria:
- [ ] Lecturer should be able to create a new or edit a poll in 'created', 'active' or 'running' status
- [ ] Lecturer should be able to start and stop a poll in 'running' or 'active' status
- [ ] Students should be able to vote in 'running' or 'active' status
- [ ] When in lecture is in 'created' status, clicking 'View' should open 'settings'-tab but all tabs should be visible

Testing steps:
- Login as a lecturer
- Create a lecture for a future date
- Create a poll
- Edit the poll
- See that starting/stopping is not possible unless day of lecture is current day

Steps needed for migration:
- none
---

Definition Of Done:
